### PR TITLE
Updated items up to 21122022

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: N/A

* **Server Mode**: renewal

* **Description of Pull Request**: 

- [x] Code review

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -167661,6 +167829,153 @@ Body:
...
+            if (.@g>=ENCHANTGRADE_B) {
+               bonus2 bAddSize,Size_Large,15;
+               if (.@g>=ENCHANTGRADE_A) {
+                  bonus bPAtk,5;
+               }
+            }
+         }
+      }
+  - Id: 510104
+    AegisName: aegis_510104
```
Comment (fab10Carvalho) [inline db/re/item_db_equip.yml:167911]: `AegisName: Mocadas_Degger`

File: db/re/item_combos.yml [hunk 2]
```diff
@@ -31690,3 +31690,49 @@ Body:
...
+  - Combos:
+      - Combo:
+          - aegis_300447   # 300447
+          - aegis_300449   # 300449
+    Script: |
+      bonus bLongAtkRate,8;
+  - Combos:
+      - Combo:
+          - Mocadas_Knife   # 510103
+          - aegis_510104   # 510104
```
Comment (fab10Carvalho) [inline db/re/item_combos.yml:31702]: `- Mocadas_Degger   # 510104`

File: db/re/item_db_etc.yml [hunk 3]
```diff
@@ -50295,6 +50295,162 @@ Body:
     Script: |
       bonus2 bSubEle,Ele_Neutral,-10;
       bonus bLongAtkRate,5+4*(getrefine()/2);
+  - Id: 300441
+    AegisName: aegis_300441
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50299]:
```suggestion
    AegisName: Burning_Knight_Card
```

File: db/re/item_db_etc.yml [hunk 4]
```diff
@@ -50295,6 +50295,162 @@ Body:
...
+      bonus bAtkRate,5;
+      .@view = getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW);
+      if (.@view == W_1HAXE || .@view == W_2HAXE) {
+         bonus bLongAtkRate,2*(.@r/3)+10;
+         if (BaseLevel>=200) {
+            bonus bLongAtkRate,3*(.@r/3);
+         }
+      }
+  - Id: 300442
+    AegisName: aegis_300442
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50318]:
```suggestion
    AegisName: Dedsera_Card
```

File: db/re/item_db_etc.yml [hunk 5]
```diff
@@ -50295,6 +50295,162 @@ Body:
...
+      .@r = getrefine();
+      bonus bMatkRate,5;
+      if (getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW) == W_STAFF) {
+         bonus2 bMagicAtkEle,Ele_All,10+2*(.@r/3);
+      }
+      if (BaseLevel>=200) {
+         bonus2 bMagicAtkEle,Ele_All,3*(.@r/3);
+      }
+  - Id: 300443
+    AegisName: aegis_300443
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50337]:
```suggestion
    AegisName: Hardrock_Titan_Card
```

File: db/re/item_db_etc.yml [hunk 6]
```diff
@@ -50295,6 +50295,162 @@ Body:
...
+      .@r = getrefine();
+      bonus bAtkRate,5;
+      if (.@r>=7) {
+         bonus2 bAddEle,Ele_Earth,10;
+         if (.@r>=9) {
+            bonus2 bAddEle,Ele_Earth,7;
+         }
+      }
+  - Id: 300444
+    AegisName: aegis_300444
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50356]:
```suggestion
    AegisName: Deadween_Card
```

File: db/re/item_db_etc.yml [hunk 7]
```diff
@@ -50295,6 +50295,162 @@ Body:
...
+      .@r = getrefine();
+      bonus bAtkRate,5;
+      if (.@r>=7) {
+         bonus bAtkRate,3;
+         if (.@r>=9) {
+            bonus bAtkRate,2;
+         }
+      }
+  - Id: 300445
+    AegisName: aegis_300445
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50375]:
```suggestion
    AegisName: Gaia_Pol_Card
```

File: db/re/item_db_etc.yml [hunk 8]
```diff
@@ -50295,6 +50295,162 @@ Body:
...
+    Flags:
+      BuyingStore: true
+      DropEffect: CLIENT
+    Script: |
+      bonus2 bAddRace,RC_Plant,40;
+      bonus2 bAddRace,RC_DemiHuman,40;
+      bonus2 bAddEle,Ele_Neutral,40;
+      bonus2 bAddEle,Ele_Earth,40;
+  - Id: 300446
+    AegisName: aegis_300446
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50390]:
```suggestion
    AegisName: Punch_Bug_Card
```

File: db/re/item_db_etc.yml [hunk 9]
```diff
@@ -50295,6 +50295,162 @@ Body:
...
+    Script: |
+      bonus bAtkRate,5;
+      if (getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW) == W_KNUCKLE) {
+         bonus bShortAtkRate,10*(.@r/3)+2*(.@r/3);
+         if (BaseLevel>=200) {
+            bonus bShortAtkRate,3*(.@r/3);
+         }
+      }
+  - Id: 300447
+    AegisName: aegis_300447
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50408]:
```suggestion
    AegisName: Aferde_Card
```

File: db/re/item_db_etc.yml [hunk 10]
```diff
@@ -50295,6 +50295,162 @@ Body:
...
+    Name: Aperde Card    # !todo check english name
+    Type: Card
+    Weight: 10
+    Locations:
+      Left_Accessory: true
+    Flags:
+      BuyingStore: true
+      DropEffect: CLIENT
+  - Id: 300448
+    AegisName: aegis_300448
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50418]:
```suggestion
    AegisName: Dispol_Card
```

File: db/re/item_db_etc.yml [hunk 11]
```diff
@@ -50295,6 +50295,162 @@ Body:
...
+    Locations:
+      Left_Accessory: true
+    Flags:
+      BuyingStore: true
+      DropEffect: CLIENT
+    Script: |
+      bonus2 bMagicAtkEle,Ele_Holy,35;
+      bonus bUseSPrate,10;
+  - Id: 300449
+    AegisName: aegis_300449
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50431]:
```suggestion
    AegisName: Timbers_Card
```

File: db/re/item_db_etc.yml [hunk 12]
```diff
@@ -50295,6 +50295,162 @@ Body:
...
+    Weight: 10
+    Locations:
+      Right_Accessory: true
+    Flags:
+      BuyingStore: true
+      DropEffect: CLIENT
+    Script: |
+      bonus bLongAtkRate,6;
+  - Id: 300450
+    AegisName: aegis_300450
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:50443]:
```suggestion
    AegisName: Renire_Card
```

File: db/re/item_db_etc.yml [hunk 13]
```diff
@@ -70253,6 +70465,57 @@ Body:
     Weight: 10
     Flags:
       BuyingStore: true
+  - Id: 1001198
+    AegisName: aegis_1001198
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:70469]:
```suggestion
    AegisName: Sources_Of_Pollution
```

File: db/re/item_db_etc.yml [hunk 14]
```diff
@@ -70253,6 +70465,57 @@ Body:
...
+    Name: Source of contamination    # !todo check english name
+    Type: Etc
+    Weight: 10
+    Flags:
+      BuyingStore: true
+      DropEffect: CLIENT
+  - Id: 1001199
+    AegisName: aegis_1001199
```
Comment (fab10Carvalho) [inline db/re/item_db_etc.yml:70477]:
```suggestion
    AegisName: Fragments_Of_Pollution
```

File: db/re/item_combos.yml [hunk 15]
```diff
@@ -31690,3 +31690,49 @@ Body:
           - Holy_Cannon_Ball    # 18001
     Script: |
       bonus2 bSubSize,Size_Small,8;
+  - Combos:
+      - Combo:
+          - aegis_300447   # 300447
```
Comment (fab10Carvalho) [inline db/re/item_combos.yml:31695]:
```suggestion
          - Aferde_Card   # 300447
```

File: db/re/item_combos.yml [hunk 16]
```diff
@@ -31690,3 +31690,49 @@ Body:
...
+          - aegis_300449   # 300449
```
Comment (fab10Carvalho) [inline db/re/item_combos.yml:31696]:
```suggestion
          - Timbers_Card   # 300449
```

**Review by google024 — COMMENTED** [review]
Thankl

**Pull request merged**